Vizzuality/half-earth-v3
# Feature/contentful translation

## Contentful translation
### Description
- Clean contentful code
- Make possible to translate contentful content by the selected language on the app

### Testing instructions
- Go to the feature globe. The content on the main card and the point card should change on translation depending on the country. Right now we only have spanish example content for 13.Amazon Basin

### Feature relevant tickets
https://vizzuality.atlassian.net/browse/HE-481
